src/submission/submission-service.ts

```diff
@@ -918,9 +918,18 @@ export namespace operations {
 				);
 				errorsAccumulator = errorsAccumulator.concat(programIdError);
 
-				// special case for therapies where we need to populate the rxnorm Ids, only do this step
-				// if the record is valid so far to avoid spending alot of time here
-				if (errorsAccumulator.length == 0 && isRxNormTherapy(command.clinicalType)) {
+				// Records using alternate Drug DB fields should skip RxNorm validation
+				// Drug DB field validation is handled downstream in validation-clinical/therapy.ts
+				const { drug_database, drug_term, drug_id } = processedRecord;
+				const hasDrugDbFields = Boolean(drug_database || drug_id || drug_term);
```

Comment:
This allows me to skip rxNormDB validation if I provide any one of these three properties.

Do you have a check elsewhere that ensures if I have one of these fields that i have all of these and none of the rxnorm fields?

Reply:
This is handled by a mix of Dictionary and Clinical validations

Dictionary enforces that either RxNorm or Drug Db are submitted: 
https://github.com/icgc-argo/argo-dictionary/pull/437/files#diff-893b20bc0b00b45e9cb1df1d1a13b5cba5dcf4627f2f182f345762618077ac54

Clinical enforces that all 3 Drug Db fields are submitted when RxNorm is blank:
https://github.com/icgc-argo/argo-clinical/pull/1195/files#diff-57392d148ba45d8530fa1789680e51f9398e3b20e910bccf1fa0e95b78445a08R297